devel/fastmod/Portfile

```
@@ -0,0 +1,125 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cargo   1.0
+PortGroup               github  1.0
+
+github.setup            facebookincubator fastmod 0.4.3 v
+github.tarball_from     archive
+
+categories              devel textproc
+platforms               darwin
+license                 Apache-2
+maintainers             {@cideM gmail.com:fbeeres}
```

Most ports should also be `openmaintainer` unless there is a specific reason why only you and you alone should be the only one to maintain/modify the port:

```suggestion
maintainers             {@cideM gmail.com:fbeeres} openmaintainer
```